packages/extension/src/shared/token/storage.ts

```diff
@@ -96,7 +101,7 @@ export async function updateTokenListNow() {
 
 export async function getTokenList(): Promise<Token[]> {
   const tokenListTokens = await tokenListStore.get()
-  return tokenListTokens.tokens.map((t) => ({ ...t, verified: true }))
+  return tokenListTokens.tokens
```

Comment:
Question: Why do we need to remove the `verified: true`?

Reply:
Because we have set `verified` to `true` when we store them [here](https://github.com/alephium/extension-wallet/commit/7c886fc686f6d6561bc4f663d84db1dd8f0ff5cd#diff-1b16e7e6c657604104f15f1fb5f9e6ec8081261573864bd04285f3f8bd4703d9R34)